habitat_baselines/common/utils.py

```diff
@@ -174,3 +226,92 @@ def generate_video(
         tb_writer.add_video_from_np_images(
             f"episode{episode_id}", checkpoint_idx, images, fps=fps
         )
+
+
+def image_resize_shortest_edge(img, size: int, channels_last: bool = False):
```

Comment:
Please, specify type for img.
```suggestion
def image_resize_shortest_edge(img: torch.Tensor, size: int, channels_last: bool = False):
```